app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command, file_to_opening, path = command.split()
+    if command == "mv":
+        if path.count("/"):
+            path_to_writing = path.split("/")
```

Comment:
Make your code universal, use os.path.split(). For example, windows use `\` instead of `/`

Reply:
Done